### PR TITLE
[TW-1208] Public api top trending

### PR DESCRIPTION
**Pull request by petewitcher-postman**
Updated images and text to capture UI drift. The new trending sections don't affect the content; they're covered by existing references to popular and trending APIs.

### CODE REVIEW

File: src/pages/docs/getting-started/exploring-public-api-network.md [hunk 1]
```diff
@@ -35,16 +35,26 @@ The Postman Public API Network is the largest network of APIs, workspaces, and c
...
-* **In the spotlight** highlights the latest exciting, useful, and noteworthy contributions to the Public API Network, handpicked by the experts at Postman.
-* **Trending workspaces, collections, and APIs** shows entities that are the most popular in the Public API Network, based on the number of people viewing or creating forks of them.
-* The **Search Postman** box at the top of the page searches all Postman teams, workspaces, collections, and APIs by default, but you can also use the search controls to limit a search to the Public API Network.
-* Under **Browse**, you can select to view **Teams**, **Workspaces**, **APIs**, and **Collections**. You can sort results by **Featured**, **Most Viewed**, and **Latest** using the **Sort by** dropdown list.
-    * You can also filter Collections, APIs, and Workspaces using the categories listed on their results pages.
+* **Spotlight** highlights the latest exciting, useful, and noteworthy contributions to the Public API Network, handpicked by the experts at Postman.
```
Comment (nerminamiller-postman) [inline src/pages/docs/getting-started/exploring-public-api-network.md:42]:
```suggestion
* **Spotlight** highlights the latest contributions to the Public API Network curated by Postman experts.
```

**Review by nerminamiller-postman — APPROVED** [review]
Just a bit of style feedback

File: src/pages/docs/getting-started/exploring-public-api-network.md [hunk 2]
```diff
@@ -35,16 +35,26 @@ The Postman Public API Network is the largest network of APIs, workspaces, and c
 
 To access the Public API Network, visit [postman.com/explore](https://www.postman.com/explore) or select **Explore** in the header.
 
-<img alt="Postman Public API Network" src="https://assets.postman.com/postman-docs/v10/public-api-network-v10.jpg" width="1000px"/>
+<img alt="Postman Public API Network" src="https://assets.postman.com/postman-docs/v10/public-api-network-v10-1.jpg" width="1000px"/>
```
Comment (mark-anthony-postman) [inline src/pages/docs/getting-started/exploring-public-api-network.md:38]:
```suggestion
<img alt="Postman Public API Network" src="https://assets.postman.com/postman-docs/v10/public-api-network-v10-1.jpg" />
```

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

**Review by gitThingsDone — APPROVED** [review]
LGTM

**Review by jkonrath-postman — APPROVED** [review]
LGTM

**Pull request merged**